package exception;

public class Example4 {

	public static void main(String[] args) {
		
		int arr[] = {20, 30, 40, 5, 17};
		
		System.out.println(arr[30]); // 5
		
		System.out.println("main method end..");
	}
}
